converter/src/main/java/gov/cms/qpp/conversion/model/TemplateId.java

```diff
@@ -126,18 +134,23 @@ public String getTemplateId(Context context) {
 	 * @param extension The extension part of the templateId.
 	 * @return The template ID if found.  Else {@code TemplateId.DEFAULT}.
 	 */
-	public static TemplateId getTemplateId(String root, String extension, Context context) {
+	public static TemplateId getTemplateId(final String root, final String extension, final Context context) {
 		Map<String, TemplateId> extensionsToTemplateId = ROOT_AND_TO_TEMPLATE_ID.get(root);
+		Function<Boolean, TemplateId> templateIdFunction = (condition) -> condition ?
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "condition" parameter (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)